pkg/controller/lifecycle/actuator.go

```diff
@@ -298,6 +323,28 @@ func (a *actuator) deleteSeedResources(ctx context.Context, ex *extensionsv1alph
 	return nil
 }
 
+func (a *actuator) cleanupShootDNSEntries(ctx context.Context, ex *extensionsv1alpha1.Extension) error {
+	cluster, err := controller.GetCluster(ctx, a.Client(), ex.Namespace)
+	if err != nil {
+		return err
+	}
+	err = a.createSeedResources(ctx, cluster, ex, false, false)
```

Comment:
may be this function now should be better named `updateSeedResources`.

Reply:
renamed to `createOrUpdateResources`